stdlib/contextlib.pyi

```diff
@@ -23,17 +23,19 @@ _T = TypeVar("_T")
 _T_co = TypeVar("_T_co", covariant=True)
 _T_io = TypeVar("_T_io", bound=Optional[IO[str]])
 _F = TypeVar("_F", bound=Callable[..., Any])
+_P = ParamSpec("_P")
 
 _ExitFunc = Callable[[Optional[Type[BaseException]], Optional[BaseException], Optional[TracebackType]], bool]
 _CM_EF = TypeVar("_CM_EF", ContextManager[Any], _ExitFunc)
 
 class _GeneratorContextManager(ContextManager[_T_co]):
     def __call__(self, func: _F) -> _F: ...
 
-def contextmanager(func: Callable[..., Iterator[_T]]) -> Callable[..., _GeneratorContextManager[_T]]: ...
+# type: ignore to deal with incomplete ParamSpec support in mypy
```

Comment:
This might throw mypy, even if it complains about line 35. Just remove the ":" maybe.

Reply:
Oops, you're right